wagtail/admin/templates/wagtailadmin/shared/side_panel_toggles.html

```diff
@@ -2,7 +2,7 @@
 
 {% for panel in side_panels %}
     <button type="button"
-        class="{{ nav_icon_button_classes }} expanded:w-text-text-label"
+        class="{{ nav_icon_button_classes }}"
```

Comment:
Note this `expanded:w-text-text-label` and other "expanded" styles are now applied to all slim header "nav icon buttons", which includes the link to the History page. The classes do nothing on those `a` tags but I thought we might as well add them there for simplicity.